Dockerfile

```diff
@@ -3,11 +3,14 @@ FROM quay.io/ukhomeofficedigital/nodejs-base:v4.4.2
 RUN mkdir /public
 RUN yum clean all && \
   yum update -y -q && \
-  yum install -y -q git && \
   yum clean all && \
   rpm --rebuilddb && \
-  npm install -g npm@latest --loglevel warn
+  npm --loglevel warn install -g npm@3
 
+COPY package.json /app/package.json
+RUN npm cache clean
+RUN npm --loglevel warn install --production --no-optional
 COPY . /app
+RUN npm --loglevel warn run postinstall
```

Comment:
This is what's failing the build. You're calling `postinstall` but no such script exists. What are you trying to do?